source/Octopus.Tentacle.Tests.Integration/ClientScriptExecutionRetriesTimeout.cs

```diff
@@ -276,21 +261,21 @@ public async Task WhenRpcRetriesTimeOut_DuringGetStatus_TheRpcCallIsCancelled(Te
                 .WithPortForwarderDataLogging()
                 .WithResponseMessageTcpKiller(out var responseMessageTcpKiller)
                 .WithPortForwarder(out var portForwarder)
+                .WithTcpConnectionUtilities(Logger, out var tcpConnectionUtilities)
                 .WithTentacleServiceDecorator(new TentacleServiceDecoratorBuilder()
-                    .LogCallsToScriptServiceV2()
-                    .CountCallsToScriptServiceV2(out var scriptServiceCallCounts)
-                    .RecordExceptionThrownInScriptServiceV2(out var scriptServiceV2Exception)
-                    .DecorateScriptServiceV2With(d => d
-                        .BeforeGetStatus(async (service, _) =>
+                    .RecordMethodUsages(tentacleConfigurationTestCase, out var recordedUsages)
+                    .HookServiceMethod(tentacleConfigurationTestCase,
+                        nameof(IAsyncClientScriptServiceV2.GetStatusAsync),
+                        async (_, _) =>
                         {
-                            // Kill the first GetStatus call to force the rpc call into retries
-                            if (scriptServiceV2Exception.GetStatusLatestException == null)
+                            // Kill the first StartScript call to force the rpc call into retries
```

Comment:
Should this still be `GetStatus` in the comment?

Reply:
Or maybe the test name needs to be updated?

Reply:
Same for the rest of these comments in this file.

Reply:
might be a copy pasta issue, let me check

Reply:
Yeah, was just a copy-paste issue